src/cmdline/mod.rs

```diff
@@ -79,7 +85,16 @@ fn valid_str(s: &str) -> Result<()> {
     }
 }
 
-fn valid_element(s: &str) -> Result<()> {
+fn is_quoted(s: &str) -> bool {
+    if let Some(first_char) = s.chars().next() {
+        if let Some(last_char) = s.chars().last() {
+            return first_char == '"' && last_char == '"';
+        }
+    }
+    false
+}
```

Comment:
IMHO this implementation would accept input with only a '"' character, should we check if `next()` and `last()` actually pointing different location 🤔, or else we check the length of `s` which should be at least 2 characters long

Reply:
Hi, I've updated this function as per your suggestion. Can you help me to review it again, please?